content/en/docs/tutorials/hello-minikube.md

```diff
@@ -90,14 +90,17 @@ tutorial has only one Container. A Kubernetes
 Pod and restarts the Pod's Container if it terminates. Deployments are the
 recommended way to manage the creation and scaling of Pods.
 
+1. Katacoda environment only: At the top of the terminal pane, click the plus sign, and then click open a new terminal.
```

Comment:
Isn't Katacoda shut down? Is there some other way to run Katacoda?

Reply:
Katacoda is working still. 
The following is what happened to me:
At [the selection of create a deployment](https://kubernetes.io/docs/tutorials/hello-minikube/#create-a-deployment), user's terminal will be block on `$ minikube dashboard`, so we need reminder users to open a new terminal.